features/step_definitions/page_content_steps.rb

```diff
@@ -0,0 +1,53 @@
+Then(/^I should see "(.+)"$/) do |text|
```

Comment:
I wonder if we should separate these out into files related to specific pages -- as of right now, I am not sure when I would look in here vs other step files. we are checking for page content all over the place